specification/marketplace/resource-manager/Microsoft.Marketplace/stable/2021-12-01/Marketplace.json

```diff
@@ -0,0 +1,2894 @@
+          "type": "integer",
+          "format": "int64",
+          "readOnly": true
+        },
+        "icon": {
+          "description": "Gets the icon url",
+          "type": "string",
+          "readOnly": true
+        },
+        "plans": {
+          "description": "Gets removed plans notifications",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/PlanNotificationDetails"
+          },
+          "readOnly": true
+        },
+        "publicContext": {
+          "description": "Gets if the offer has public plans",
+          "type": "boolean",
+          "readOnly": true
+        },
+        "subscriptionsContext": {
+          "description": "Gets the subscriptions context for private plans",
+          "type": "array",
+          "items": {
+            "type": "string"
+          },
+          "readOnly": true
+        }
+      }
+    },
+    "StopSellPayload": {
+      "description": "Private plans subscriptions",
+      "type": "object",
+      "properties": {
+        "subscriptions": {
+          "type": "array",
+          "items": {
+            "type": "string"
+          }
+        }
+      }
+    },
+    "SubscriptionsResponse": {
+      "description": "Subscription list operation response.",
+      "type": "object",
+      "properties": {
+        "value": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/Subscription"
+          },
+          "description": "An array of subscriptions.",
+          "readOnly": true
+        },
+        "skipToken": {
+          "type": "string",
+          "description": "The skip token to get the next page.",
+          "readOnly": true
+        },
+        "count": {
+          "description": "Gets the number of subscriptions on a page",
+          "type": "integer",
+          "format": "int64",
+          "readOnly": true
+        }
+      }
+    },
+    "Subscription": {
+      "description": "Subscription information.",
+      "type": "object",
+      "properties": {
+        "id": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The fully qualified ID for the subscription. For example, /subscriptions/00000000-0000-0000-0000-000000000000."
+        },
+        "subscriptionId": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The subscription ID."
+        },
+        "displayName": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The subscription display name."
+        },
+        "state": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The subscription state. Possible values are Enabled, Warned, PastDue, Disabled, and Deleted.",
+          "enum": [
+            "Enabled",
+            "Warned",
+            "PastDue",
+            "Disabled",
+            "Deleted"
+          ],
+          "x-ms-enum": {
```

Comment:
>x-ms-enum [](http://example.com/codeflow?start=11&length=9)

You can add a description for each enum value as well.

Reply:
I think that the enum values are self-explained ( and I copied it as is from Subscriptions RP documentation)